dashboard/app/controllers/plc/enrollment_task_assignments_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Plc::EnrollmentTaskAssignmentsController < ApplicationController
+  before_action :set_plc_enrollment_task_assignment, only: [:show, :destroy]
+
+  # GET /plc/enrollment_task_assignments
+  # GET /plc/enrollment_task_assignments.json
+  def index
+    @plc_enrollment_task_assignments = Plc::EnrollmentTaskAssignment.all
+  end
+
+  # GET /plc/enrollment_task_assignments/1
+  # GET /plc/enrollment_task_assignments/1.json
+  def show
+  end
+
+  # DELETE /plc/enrollment_task_assignments/1
+  # DELETE /plc/enrollment_task_assignments/1.json
+  def destroy
+    @plc_enrollment_task_assignment.destroy
+    respond_to do |format|
+      format.html { redirect_to plc_enrollment_task_assignments_url }
+      format.json { head :no_content }
+    end
```

Comment:
(can't seem to comment in response to everything)

I'm taking out new enrollment task assignment creation. It's not necessary for this stage, or really to ever be shown to a user.

Removed format boilerplate